src/gp/src/GPMSAOptions.C

```diff
@@ -558,6 +594,30 @@ GPMSAOptions::set_final_scaling
           }
     }
 
+  if (m_autoscaleMinMaxAll || !m_autoscaleMinMaxOutput.empty())
+    {
+      V maxOutput(*m_simulationOutputs[0]);
+
+      V minOutput(*m_simulationOutputs[0]);
+
+      for (unsigned int i=1; i < m_simulationOutputs.size(); ++i)
+        min_max_update(minOutput, maxOutput,
+                       (*m_simulationOutputs[i]));
+
+      for (unsigned int p=0; p != dimOutput; ++p)
+        if (m_autoscaleMinMaxAll ||
+            m_autoscaleMinMaxOutput.count(p))
+          {
+            if ((m_outputScaleMin.size() > p) &&
+                ((m_outputScaleMin[p] != 0) ||
```

Comment:
Nah.  The fill in set_output_scaling is with *exactly* 0 and 1, so we can rely on an exact test to tell us if something other than a fill happened.

Now, there is a loophole here: if someone manually sets 0 and 1, then requests automatic scaling, then we won't catch that and scream at them - by doing something insane *twice* they'll have evaded our sanity check...